cmd/loadtest/app.go

```diff
@@ -254,6 +256,8 @@ inscription - sending inscription transactions`)
 	ltp.RecallLength = LoadtestCmd.Flags().Uint64("recall-blocks", 50, "The number of blocks that we'll attempt to fetch for recall")
 	ltp.ContractAddress = LoadtestCmd.Flags().String("contract-address", "", "The address of the contract that will be used in `--mode contract-call`. This must be paired up with `--mode contract-call` and `--calldata`")
 	ltp.ContractCallData = LoadtestCmd.Flags().String("calldata", "", "The hex encoded calldata passed in. The format is function signature + arguments encoded together. This must be paired up with `--mode contract-call` and `--contract-address`")
+	ltp.ContractCallFunctionSignature = LoadtestCmd.Flags().String("function-signature", "", "The contract's function signature that will be called. The format is '<function name>(<types...>)'. This must be paired up with `--mode contract-call` and `--contract-address`. If the function requires parameters you can pass them with `--function-arg <value>`.")
```

Comment:
Not important but make sure to remove the backticks from descriptions (see the [related](https://polygon.atlassian.net/browse/DVT-1166) issue).